api/src/controllers/homepage.js

```diff
@@ -18,9 +19,7 @@ async function getHomepage(req, res, next) {
     if (foundHomepage) {
       return res.status(200).json(foundHomepage);
     }
-    // note: each string in `aboutUs` is a paragraph
-    const emptyHomepage = { teamId: team_id, aboutUs: [''] };
-    return res.status(200).json(emptyHomepage);
+    return next(fillErrorObject(404, 'Validation error', ['No homepage found with the given team_id']));
```

Comment:
Thanks for writing those triggers! I can imagine one downside where, if the trigger is somehow removed or if we migrate dbs, we can run into problems when we run our code. I'd add documentation somewhere to note that we have a trigger to updated collection x based on insertions to the `Team` model. Maybe the best place is to add a comment in the schema model files of affected collections, like `Homepage`, `Website`, etc.